docs/source/user-guide/tasks/build-packages/compiler-tools.rst

```diff
@@ -338,3 +338,81 @@ This example is a little contrived, because the ``m2w64-gcc_win-64`` package is
+``pkg-config`` executable and returning the result. The code for this can be seen at
+https://github.com/AnacondaRecipes/pkg-config-feedstock/blob/master/recipe/pkg-config
+
+Qt (macOS)
+----------
+
+The ``Anaconda Distribution`` does not require installing ``Xcode``. The entire distribution can be built from
+source without it. Unfortunately ``Qt`` hard-codes the use of ``/usr/bin/xcodebuild`` and ``/usr/bin/xcrun``.
+We have changed this in patches to the source for our ``Qt`` packages (https://github.com/AnacondaRecipes/qt-feedstock/blob/master/recipe/0001-qtwebengine-allow-any-xcblah-in-PATH.patch
+and https://github.com/AnacondaRecipes/qt-feedstock/blob/master/recipe/0011-osx-allow-any-xcrun-in-PATH.patch)
+so that ``xcodebuild`` and ``xcrun`` are used instead (and ``PATH`` lookup finds them). For this to work, we
+have created fake ``xcodebuild`` and ``xcrun`` scripts (https://github.com/AnacondaRecipes/qt-feedstock/blob/master/recipe/xcodebuild and https://github.com/AnacondaRecipes/qt-feedstock/blob/master/recipe/xcrun)
+that return values needed to achieve compatibiltiy with the ``Anaconda Distribution``. These scripts cannot
+live in ``${CONDA_PREFIX}/bin`` without risking disruption to ``Xcode`` users, therefore they are installed
+in ``${CONDA_PREFIX}/bin/xc-avoidance``. To use them you should add this to ``PATH`` in ``build.sh``. An
+example of this can be seen at https://github.com/AnacondaRecipes/pyqt-feedstock/blob/master/recipe/build.sh#L8
+
+libtool
+-------
+
+There is a feature in various linkers which is most commonly called 'as-needed'. When the linker detects that
+no symbols are referenced in a consumer executable (exe) or dynamically shared objects (DSO) it elides this
+``DT_NEEDED`` entry from the ``ELF dynamic section``. We do this so that our software loads more quickly. It
+also leads to the possibility of implementing a new feature in ``conda-build`` whereby it would warn when a
+'library' package (identified via some heuristics, a ``lib`` prefix perhaps; existence of DSOs?) ends up in
+the runtime requirements but is not actually used in any of the exes or DSOs. This would allow us to consider
+these as candidates for removal thus making the package less heavy-weight. For the curious, there's a function
+in `conda-build` called `check_overlinking` and the original intention for this function was to do this but
+it ended up checking for the opposite (worse!) problem of `underlinking`. This we define as missing
+dependencies, which happens frequently when something gets installed in the ``host prefix`` through a
+transitive dependency and is therefore not listed as a direct run dependency despite being directly linked to
+some exe(s) and/or DSO(s) in the package (build systems are sometimes 'greedy' linking to whatever they can
+find). We denote this as `worse` because when someone installs these two packages (3 including the dependency)
```

Comment:
```suggestion
find). We denote this as 'worse' because when someone installs these two packages (3 including the dependency)
```

Reply:
@mingwandroid I think this one has still the wrong highlighting

Reply:
I don't get this feature! I  had to `Resolve conversation` then uresolve it again before I could apply your suggestion. Oh well, I think it worked.